packages/shared/lib/services/sync/orchestrator.service.ts

```diff
@@ -192,6 +199,14 @@ export class Orchestrator {
             environment_id: environmentId
         };
         const activityLogId = await createActivityLog(log);
```

Comment:
is it okay I had this in the sentence instead of changing the command? (e.g: `Sync was updated with command: "trigger sync" (full)`)
because creating a new command will have a much bigger impact

Reply:
nvm let's use RUN_FULL then